### ISSUE TITLE
File menu: all items should have access keys

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#6188]
### Steps to reproduce

Press Alt + F to open the File menu.

### Expected behavior

All the items on the menu have access keys

### Actual behavior

These items are missing access keys:
Open from cloud
Save to cloud
Update Cloud audio preview
Export other
Share audio

### Audacity Version

latest beta

### Operating system

Windows
